lib/bibliothecary/multi_parsers/dependencies_csv.rb

```diff
@@ -0,0 +1,151 @@
+require 'csv'
+
+module Bibliothecary
+  module MultiParsers
+    module DependenciesCSV
+      include Bibliothecary::Analyser
+      include Bibliothecary::Analyser::TryCache
+
+      def self.mapping
+        {
+          match_filename('dependencies.csv') => {
+            kind: 'lockfile',
+            parser: :parse_dependencies_csv
+          }
+        }
+      end
+
+      # Processing a CSV file isn't as exact as using a real manifest file,
+      # but you can get pretty close as long as the data you're importing
+      # is simple.
+      class CSVFile
+        # Header structures are:
+        #
+        # <field to fill in for dependency> => {
+        #   match: [<regexp of incoming column name to match in priority order>...],
+        #   [default]: <optional default value for this field>
+        # }
+        HEADERS = {
+          "platform" => {
+            match: [
+              /^platform$/i
+            ]
+          },
+          "name" => {
+            match: [
+              /^name$/i
+            ]
+          },
+          # The "requirement" column in this case is going to want exact
+          # versions. Prefer those over requirements, and don't use the
+          # Lockfile Requirement column from CSV exports.
```

Comment:
I'm sort of not understanding this comment but I'm not sure whether it's a substantive concern or me just not understanding what you mean.

 * checking an assumption, I think in the export "manifest_requirement" is the package.json requirement expresson, and "lockfile_requirement" is the actual version from package-lock.json?
 * If so and we're going to return kind=lockfile here, I'd imagine us preferring the exact version?